lib/httplib/csrf/csrf.go

```diff
@@ -35,7 +35,14 @@ import (
 // It's prefixed with "__Host-" as an additional defense in depth measure.
 // This makes sure the cookie is sent from a secure page (HTTPS),
 // won't be sent to subdomains, and the path attribute is set to /.
```

Comment:
This godoc is associated with `CookieName`. Now that you've restored the const block you need to move godoc inside.